Workbooks/AKS/Syslog/Syslog.workbook

```diff
@@ -1,568 +1,561 @@
+								},
+								{
+									"durationMs": 43200000
+								},
+								{
+									"durationMs": 86400000
+								},
+								{
+									"durationMs": 172800000
+								},
+								{
+									"durationMs": 259200000
+								},
+								{
+									"durationMs": 604800000
+								},
+								{
+									"durationMs": 1209600000
+								},
+								{
+									"durationMs": 2419200000
+								},
+								{
+									"durationMs": 2592000000
+								},
+								{
+									"durationMs": 5184000000
+								},
+								{
+									"durationMs": 7776000000
+								}
+							]
+						},
+						"value": {
+							"durationMs": 43200000
+						},
+						"label": "Time Range"
+					},
+					{
+						"id": "0698db8c-7a3a-4aec-bfb2-eb59942b0375",
+						"version": "KqlParameterItem/1.0",
+						"name": "Workspace",
+						"type": 5,
+						"isRequired": true,
+						"multiSelect": true,
+						"quote": "'",
+						"delimiter": ",",
+						"query": "resources\r\n| where type =~ '{ResourceType}'\r\n| where name =~ \"{Resource:Name}\"\r\n| extend workspaceResourceId = tostring(parse_json(tolower(properties)).addonprofiles.omsagent.config.loganalyticsworkspaceresourceid)\r\n| project workspaceResourceId ",
+						"crossComponentResources": [
+							"{Resource}"
+						],
+						"isHiddenWhenLocked": true,
+						"typeSettings": {
+							"additionalResourceOptions": [
+								"value::1"
+							],
+							"showDefault": false
+						},
+						"defaultValue": "value::1",
+						"queryType": 1,
+						"resourceType": "microsoft.containerservice/managedclusters"
+					},
+					{
+						"id": "9d866cc4-c583-4876-8213-6193548cbca1",
+						"version": "KqlParameterItem/1.0",
+						"name": "HostName",
+						"type": 2,
+						"isRequired": true,
+						"multiSelect": true,
+						"quote": "'",
+						"delimiter": ",",
+						"query": "let T = (Syslog | distinct HostName );\r\nlet T_has_records = toscalar(Syslog | summarize count() > 0);\r\nunion \r\n(T | where T_has_records == true), (datatable(HostName:string)['HostName'] | where T_has_records == false)\r\n| distinct HostName",
+						"crossComponentResources": [
+							"{Workspace}"
+						],
+						"typeSettings": {
+							"additionalResourceOptions": [
+								"value::all"
+							],
+							"showDefault": false
+						},
+						"defaultValue": "value::all",
+						"queryType": 0,
+						"resourceType": "microsoft.operationalinsights/workspaces",
+						"label": "Node"
+					},
+					{
+						"id": "1119cb83-4bd5-4b22-a20e-f54383f0e240",
+						"version": "KqlParameterItem/1.0",
+						"name": "Facility",
+						"type": 2,
+						"isRequired": true,
+						"multiSelect": true,
+						"quote": "'",
+						"delimiter": ",",
+						"query": "let T = (Syslog | distinct Facility );\r\nlet T_has_records = toscalar(Syslog | summarize count() > 0);\r\nunion \r\n(T | where T_has_records == true), (datatable(Facility:string)['Facility'] | where T_has_records == false)\r\n| distinct Facility",

```

Comment:
Can we add the following lines to the Facility and Severity Parameter? So that they only display the values that are available during the time range. But if you have different opinions, we can discuss too.

`"timeContextFromParameter": "timeRange",`